ci-operator/templates/build-azure-image.yaml

```diff
@@ -0,0 +1,315 @@
+# This template is being used for testing OpenShift Ansible image builds for Azure.
```

Comment:
This template needs to be moved in `ci-operator/templates/openshift/openshift-azure`.